tools/noise/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+project(noise-tools C)
+
+find_library(M_LIBRARY m)
```

Comment:
You can just add "m" to the target_link_libraries call. dt proper does the same, so it should work here, too.